src/kits/kit.ts

```diff
@@ -1251,7 +1262,14 @@ export async function descriptionForKit(kit: Kit, shortVsName: boolean = false):
     if (kit.name === SpecialKits.ScanForKits) {
         return localize('search.for.compilers', 'Search for compilers on this computer');
     }
-    return localize('unspecified.let.cmake.guess', 'Unspecified (Let CMake guess what compilers and environment to use)');
+    if (kit.name === SpecialKits.Unspecified) {
+        return localize('unspecified.let.cmake.guess', 'Unspecified (Let CMake guess what compilers and environment to use)');
+    }
+    if (kit.name === SpecialKits.ScanSpecificDir) {
+        return localize('search.for.compilers.in.dir', 'Search for compilers in a specific directory');
```

Comment:
We should likely make note here or surface to the user that it will recursively search, if that's what you'd like to happen.

Reply:
Updated for you .